ios/TestApp/TestAppTests/TestAppTests.mm

```diff
@@ -0,0 +1,38 @@
+#import <XCTest/XCTest.h>
+#import <torch/script.h>
+
+@interface TestAppTests : XCTestCase
+
+@end
+
+@implementation TestAppTests {
+  torch::jit::script::Module _module;
+}
+
+- (void)setUp {
+  [super setUp];
+  NSString* modelPath = [[NSBundle bundleForClass:[self class]] pathForResource:@"model"
+                                                                         ofType:@"pt"];
+  XCTAssertTrue([[NSFileManager defaultManager] fileExistsAtPath:modelPath],
+                @"model.pt doesn't exist!");
+  auto qengines = at::globalContext().supportedQEngines();
+  if (std::find(qengines.begin(), qengines.end(), at::QEngine::QNNPACK) != qengines.end()) {
+    at::globalContext().setQEngine(at::QEngine::QNNPACK);
+  }
+  torch::autograd::AutoGradMode guard(false);
```

Comment:
qq: wonder why this is needed?

Reply:
I think this is a global switch to disable Auto grad.